Configure

```diff
@@ -1451,6 +1451,13 @@ if (!$disabled{asm}) {
     }
 }
 
+if (!$disabled{ec_nistp_64_gcc_128} && !$predefined{__SIZEOF_INT128__}) {
+    die <<"_____";
+Your compiler doesn't appear to support 128-bit integer types.  Enabling
+ec_nistp_64_gcc_128 is therefore not permitted.
```

Comment:
This is potentially misleading message. Because it doesn't clarify that "appear" refers to "according to **our** book". I mean as it stands user would be entitled to exclaim "what the heck! I know it does". Maybe writing exactly as it would be more appropriate. Namely "`Your compiler doesn't pre-define __SIZEOF_INT128__`".

Reply:
Sure